pkg/services/accesscontrol/types.go

```diff
@@ -0,0 +1,26 @@
+package accesscontrol
```

Comment:
Should I move that somewhere else?

Reply:
I'd say we can put this in the `roles.go`, as the additions are serving to working with roles. Wdyt?

Reply:
Alright!